projects/1.18/assets/botania/botania/lang/zh_cn.json

```
@@ -1743,6 +1880,9 @@
   "botania.page.spreader4": "魔力发射器每次只发射一波脉冲，直到上一波命中目标才会发射下一波。$(p)此外，一段时间后脉冲也会受$(thing)魔力流失$(0)的影响。在光线中变稀薄的部分就开始发生流失了。当光线进入一个发射器时也会有少量$(thing)魔力流失$(0)。",
   "botania.page.spreader11": "提供一个$(thing)红石信号$(0)给魔力发射器会让它停止工作。在需要更好的控制时推荐$(item)红石魔力$(0)$(item)发射器$(0)，因为它可以在没有目标方块的情况经由红石脉冲激活发射脉冲。",
   "botania.page.spreader5": "合成$(item)魔力发射器$(0)",
+  "botania.page.spreader10": "用一块$(item)羊毛$(0)右击发射器便可将它套住。不仅看上去更美观，这样也会降低脉冲的噪音。$(p)再次空手右击则会移除羊毛。",
+  "botania.page.spreader12": "同样地，使用一个$(item)脚手架$(0)右键单击发射器会将其包围在一个坚固的框架中。这使得它可以支持脆弱的红石组件，如按钮和拉杆的可以安装到任意面。红石粉、红石中继器和红石比较器可以放在它上面。这似乎也允许发射器含水放置。$(p)再次空手右击则会移除脚手架。",
+
   "botania.entry.pool": "魔力池",
   "botania.tagline.pool": "魔力存储区",
   "botania.page.pool0": "$(item)魔力池$(0)简单说就是储存$(thing)魔力$(0)的地方。$(p)魔力能通过使用$(item)魔力发射器$(0)输入，与魔力池相邻的$(item)魔力发$(0)$(item)射器$(0)会自动从中抽取魔力增加内部缓存。$(p)所有$(thing)功能花$(0)需要一个相邻的$(item)魔力池$(0)提供能量。",

```

这格式化怎么了......是原文分单词加两个格式符结果没删？
（这个可能需要大范围筛查）
```suggestion
  "botania.page.pool0": "$(item)魔力池$(0)简单说就是储存$(thing)魔力$(0)的地方。$(p)魔力能通过使用$(item)魔力发射器$(0)输入，与魔力池相邻的$(item)魔力发射器$(0)会自动从中抽取魔力增加内部缓存。$(p)所有$(thing)功能花$(0)需要一个相邻的$(item)魔力池$(0)提供能量。",
```
下面一行也是，但是在修改范围以外我点不到（并且那句话好像也不太通顺）